Composer/packages/form-dialogs/src/demo/Demo.tsx

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Confirming - do you want to check in the Demo to Composer?

Reply:
Yes, it makes future work on the extension much faster.